### PR TITLE
Feature/weekly reset

### PR DESCRIPTION
**Pull request by ajg7**
# Description

Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: config/knexfile.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
     client: 'pg',
     connection: process.env.DATABASE_URL,
     migrations: { directory: '../data/migrations' },
-    seeds: { directory: '../data/seeds' },
+    seeds: { directory: '/c/Users/ajgeb/Desktop/My_Projects/labs/story-squad-be-b/data/seeds' },
```
Comment (trevorjamesmartin) [inline config/knexfile.js:9]: now that we're running raw SQL on the reset-button, we won't need to use this absolute-path. Let's revert back to using the relative-path.

File: api/cronTasks/weeklyReset.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const cron = require('node-cron');
+const cronJob = require('./cronModel');
+
+// RWD (Saturday-Monday)
+cron.schedule('* * * * * 6,0,1', () => {
+    console.log("It is working!!!");
```
Comment (trevorjamesmartin) [inline api/cronTasks/weeklyReset.js:6]: can we tailor these console logs to reflect the job being run?  i.e. "it is working: saturday-monday"

File: .prettierrc.js [hunk 3]
```diff
@@ -3,4 +3,6 @@ module.exports = {
   tabWidth: 2,
   semi: true,
   singleQuote: true,
+  // Added this 
+  endOfLine: auto
```
Comment (trevorjamesmartin) [inline .prettierrc.js:7]: is "auto" not the default?

File: api/profile/profileRouter.js [hunk 4]
```diff
@@ -4,6 +4,8 @@ const Parents = require('../parent/parentModel');
 
 const { ops } = require('../../lib');
 
+
```
Comment (trevorjamesmartin) [inline api/profile/profileRouter.js:7]: ?

Comment (trevorjamesmartin) [reply api/profile/profileRouter.js:7]: Are 2 blank lines the only diff?

File: api/cronTasks/weeklyReset.js (hunk 2 above)
Comment (ajg7) [reply api/cronTasks/weeklyReset.js:6]: I have updated

File: .prettierrc.js (hunk 3 above)
Comment (ajg7) [reply .prettierrc.js:7]: Mine did not. Without it, there were a lot of prettier errors

File: api/profile/profileRouter.js (hunk 4 above)
Comment (ajg7) [reply api/profile/profileRouter.js:7]: Not entirely sure how that happened, but I did fix

File: .prettierrc.js (hunk 3 above)
Comment (trevorjamesmartin) [reply .prettierrc.js:7]: good catch!

**Review by trevorjamesmartin — APPROVED** [review]
Looks good!

**Review by gregory-j-wilson — APPROVED** [review]
looks good.

**Pull request merged**